doc/CLI.rst

```diff
@@ -58,10 +71,9 @@ code can propagate constants using the Pythran ConstantFolding optimization::
 
   $> pythran -e cli_foo.py -p pythran.optimizations.ConstantFolding
 
-If you want to specify the path of generated file::
+If you're brave, you can generate native modules compatible with Python3. The
+hacky (and currently only) way to do so is by running::
 
-  $> pythran cli_foo.py -o /tmp/cli_foo.so -DNDEBUG
-  $> ls /tmp/cli_foo.so
-  /tmp/cli_foo.so
+  $> `pythran-config --compiler --cflags` `python3-config --cflags --libs` -lboost_python-py34 -lgmp -lgmpxx -std=c++11 cli_foo.cpp -shared -fPIC -o cli_foo -DNDEBUG
```

Comment:
On Wed, Sep 02, 2015 at 11:58:05AM -0700, pbrunet wrote:
> >  
> > -  $> pythran cli_foo.py -o /tmp/cli_foo.so -DNDEBUG
> > -  $> ls /tmp/cli_foo.so
> > -  /tmp/cli_foo.so
> > +  $> `pythran-config --compiler --cflags` `python3-config --cflags --libs` -lboost_python-py34 -lgmp -lgmpxx -std=c++11 cli_foo.cpp -shared -fPIC -o cli_foo -DNDEBUG
> 
> gmp is not in the pythran-config?
 pythran-config is called without --libs (on purpose)